specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/server.may.send.extended.pong.control.frames/request.rpt

```diff
@@ -0,0 +1,35 @@
+connect tcp://127.0.0.1:8001
+connected
+
+# Connection request
+write "GET /echo?.kl=Y HTTP/1.1\r\n"
+write "Upgrade: websocket\r\n"
+write "Connection: Upgrade\r\n"
+write "Host: localhost:8001\r\n"
+write "Origin: http://localhost:8001\r\n"
+write "Sec-WebSocket-Key: nDaimG37f4nUqogPypithw==\r\n"
+write "Sec-WebSocket-Version: 13\r\n"
+write "Sec-WebSocket-Extensions: x-kaazing-ping-pong\r\n"
+write "\r\n"
+
+# 101 Response
+read "HTTP/1.1 101 Web Socket Protocol Handshake\r\n"
+read "Connection: Upgrade\r\n"
+read "Date:" /.*/ "\r\n"
+read "Sec-WebSocket-Accept:" /.*/ "\r\n"
+# The 4 control bytes for ping-pong extension are 0x01010102:
+read "Sec-WebSocket-Extensions: x-kaazing-ping-pong; 01010102"
+read "\r\n"
+read "Server: Kaazing Gateway\r\n"
+read "Upgrade: websocket\r\n"
+read "\r\n"
+
+# binary-encoded balancer directive - NO balance needed
+read [0x82 0x04 0xEF 0x83 0xBF 0x4E]
+
+# Expect extended PONG
+read [0x81 0x06 0x01 0x01 0x01 0x02 0x0a 0x00]
+
+# Don't respond. Server should close connection (inactivity timeout expired)
```

Comment:
The client is not expected to reply to PONG frames (see RFC 6455). You should just end the connection with a clean WS close handshake(initiated by the response script) after reading the PONG. So you'll have 
read pong
read close frame
write close frame
closed